webapp/assets/css/root-tokens.css

```
@@ -191,53 +191,70 @@
-/*  Derived so that an input's box height always follows its own font size, padding and */
-/*  border. Moved here from _styleguide-tokens.scss; the unused x-large step was dropped */
-/*  (it was derived from $input-font-size-large, not from an x-large font size). */
+  /*  Derived so that an input's box height always follows its own font size, padding and */
+  /*  border. Moved here from _styleguide-tokens.scss; the unused x-large step was dropped */
+  /*  (it was derived from $input-font-size-large, not from an x-large font size). */
   --input-border-size: var(--border-size-base);
   --input-font-size-base: var(--font-size-base);
   --input-padding-vertical: var(--space-x-small);
-  --input-height: calc(var(--input-font-size-base) * var(--line-height-base) + (var(--input-padding-vertical) + var(--input-border-size)) * 2);
+  --input-height: calc(
+    var(--input-font-size-base) * var(--line-height-base) +
```

Stylelint (17.14.0)

[error] Unexpected newline after "+" (scss/operator-no-newline-after)

(scss/operator-no-newline-after)